lib/taser-markdown-report.js

```diff
@@ -47,53 +45,36 @@ class TaserMarkdownReport {
 
 module.exports = TaserMarkdownReport;
 
-function reportHeader() {
-    return [
-        '# :zap: Taser Report :zap:',
-        ''
-    ];
-}
-
 function reportError(problem) {
     return [
-        `###: :x: Error testing ${problem}`,
-        '#### Please review the problem and submit a new pull request',
-        '(hint: your function might be returning `null` or `undefined`'
+        `### 500 (Internal Taser Error)`,
+        `There was a fatal error testing ${problem} :sob:`,
+        ''
     ];
 }
 
 function problemReportHeader(problem) {
     return [
-        '',
-        '----',
-        '',
-        `## Problem: ${problem}`
+        `## Problem: ${problem}`,
+        ''
     ];
 }
 
 function problemReportBody(report) {
     return [
         testReportHeader(report),
         testReportErrors(report.errors),
-        testReportPassed(report.testResults.passed),
-        testReportFailed(report.testResults.failed)
+        testReportFailed(report.testResults.failed),
+        testReportPassed(report.testResults.passed)
     ];
 }
 
 function testReportHeader(report) {
-    let contextName = report.browser.name;
-    let numberOfErrors = report.errors.length;
     let numberOfPasses = report.testResults.passed.length;
     let numberOfFailures = report.testResults.failed.length;
 
     return [
-        `**Browser**: ${contextName}`,
```

Comment:
Browser has gone away because it's only available in half of the test cases. Any server-side test will display "none" for this, making it unnecessary (and potentially confusing) 50% of the time